Banner

itsHackinTime opened the pull request:
I had some trouble styling buttons on the hero page. all of my styles would be overruled by the button styling from Prime React. I am also in need of an updated / better-resolution picture, and I am unsure about the font on the page. otherwise, the component is completed

tudor-seserman commented on client/src/components/home/HeroC.tsx:
```diff
@@ -1,6 +1,46 @@
+                    <br/>
+                    <div className="min-w-full flex ">
+                    <Button 
+                    className="p-hero-buttons text-xl mr-3 bg-YHighlightColor text-PrimaryColor "
```
I think the buttons should be OHighlightColor. The orangy highlight color,

tudor-seserman reviewed (changes requested):
Looks good to me. Just the buttons should be OHighlightColor to match the Design. Also the font in the design is RocknRoll One, but it looks different on the page.

itsHackinTime commented:
I didn't see a file for a "RocknRoll One" font, so I added a .ttf file to assets>custom-theme>tailwind-custom>fonts and added the @fontface in tailwind-custom>_fonts. after I made that change, most fonts were transformed to "RocknRoll One". I changed the order of ::root > font-family so "RocknRoll One" is at the back of the list. Other than that I changed the color of the buttons to OHighlightColor.
